core/chains/evm/logpoller/log_poller.go

```diff
@@ -120,7 +120,7 @@ func NewLogPoller(orm *ORM, ec Client, lggr logger.Logger, pollPeriod time.Durat
 	return &logPoller{
 		ec:                ec,
 		orm:               orm,
-		lggr:              lggr,
```

Comment:
What's the motivation for not logging the name of the service, is it redundant?

Reply:
I don't know why it didn't use to log the name of the service, it logs it now, that's what I'm trying to fix